homeassistant/components/media_player/universal.py

```diff
@@ -328,6 +332,11 @@ def current_source(self):
         return self._child_attr(ATTR_INPUT_SOURCE)
 
     @property
+    def source_list(self):
+        """List of available input sources."""
+        return self._source_list
```

Comment:
I feel that we're heading towards conflating concerns. This platform is meant to be a universal media player that wraps other media players. It is not meant to be standalone media player.